instrumentation/pubsub-1/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/pubsub/PubsubIgnoredTypesConfigurer.java

```diff
@@ -0,0 +1,17 @@
+package io.opentelemetry.javaagent.instrumentation.pubsub;
+
+import com.google.auto.service.AutoService;
+import io.opentelemetry.javaagent.extension.ignore.IgnoredTypesBuilder;
+import io.opentelemetry.javaagent.extension.ignore.IgnoredTypesConfigurer;
+import io.opentelemetry.sdk.autoconfigure.spi.ConfigProperties;
+
+@AutoService(IgnoredTypesConfigurer.class)
+public class PubsubIgnoredTypesConfigurer implements IgnoredTypesConfigurer {
+
+  @Override
+  public void configure(IgnoredTypesBuilder builder, ConfigProperties config) {
+    builder.allowClass("com.google.cloud.pubsub.")
+            .ignoreClass("com.google.api.")
+            .allowClass("com.google.api.client.http.HttpRequest");
```

Comment:
Without these 2 lines I get this error:
```
Transformed classes match global libraries ignore matcher: [com.google.api.gax.rpc.BatchedFuture, com.google.api.core.AbstractApiFuture$InternalSettableFuture, com.google.api.gax.grpc.GrpcExceptionCallable$ExceptionTransformingFuture, com.google.api.core.ListenableFutureToApiFuture, com.google.api.gax.grpc.GrpcDirectBidiStreamingCallable$2, com.google.api.gax.batching.ThresholdBatcher$1, com.google.api.gax.retrying.BasicRetryingFuture$CompletionListener, com.google.api.core.SettableApiFuture, com.google.api.gax.grpc.GrpcDirectStreamController$1, com.google.api.core.ApiFutureToListenableFuture, com.google.api.gax.retrying.BasicRetryingFuture, com.google.api.gax.retrying.CallbackChainRetryingFuture$AttemptCompletionListener, com.google.api.gax.retrying.NonCancellableFuture, com.google.api.gax.rpc.AttemptCallable, com.google.api.gax.retrying.CallbackChainRetryingFuture, com.google.api.gax.rpc.Watchdog]
```

Does anyone know if this is OK or if there's a better way?